features/step_definitions/resources.rb

```diff
@@ -125,7 +125,7 @@
 
 # tries to delete resource if it exists and make sure it disappears
 # example: I ensure "hello-openshift" pod is deleted
-Given /^(I|admin) ensures? #{QUOTED} (\w+) is deleted(?: from the#{OPT_QUOTED} project)?( after scenario)?$/ do |by, name, type, project_name, after|
+Given /^(I|admin) ensures? #{QUOTED} (.+) is deleted(?: from the#{OPT_QUOTED} project)?( after scenario)?$/ do |by, name, type, project_name, after|
```

Comment:
@akostadinov Is that ok for you?

Reply:
Better use `[\w.]+`.

Reply:
I refered to clusterservicebroker in line 218 of svc-catalog.feature. But now I looked for other examples, there are cluster_service_broker. Will use underscore in this case.

Reply:
Could you remove the change then? :)